src/_locales/pl-PL.json

```diff
@@ -0,0 +1,96 @@
+{
+  "audio-device-communications": "Domyślne urządzenie do komunikacji",
+  "audio-device-default": "Domyślne urządzenie",
+  "audio-device-unknown": "Nieznane urządzenie",
+
+  "button-test-ok": "OK",
+  "button-text-close": "Zamknij",
+  "button-text-dont-tell-me-again": "Nie pokazuj więcej",
+  "button-text-lets-do-this": "Zróbmy to!!",
+  "button-text-lets-go": "Zaczynajmy!!",
+  "button-text-not-now": "Nie teraz",
+
+  "label-about": "Informacje",
+  "label-desktop-settings": "Ustawienia Aplikacji",
+  "label-donate": "Wspomóż projekt",
+  "label-help": "Pomoc",
+  "label-issues": "Problemy",
+  "label-learn-more": "Dowiedz się więcej",
+  "label-loading-devices": "Wczytywanie urządzeń...",
+  "label-pause-after-song": "Wstrzymaj po tym utworze",
+  "label-quit": "Wyjdź",
+  "label-show-lyrics": "Pokaż tekst utworu (Beta)",
+  "label-unknown-artist": "Nieznany Wykonawca",
+  "label-unknown-song": "Nieznany Utwór",
+  "label-version": "Wersja",
+
+  "lastfm-login-authorized": "Zalogowany",
+  "lastfm-login-authorizing": "W trakcie logowania",
+  "lastfm-login-button-text": "Zaloguj się do Last.FM",  
+  "lastfm-login-error": "Wygląda na to, że przerwałeś proces logowania - jeśli to pomyłka, spróbuj jeszcze raz",
+  "lastfm-login-not-authorized": "Nie zalogowany",
+  "lastfm-logout-button-text": "Wyloguj się z Last.FM",
+
+  "lyrics-failed-message": "Niestety nie znaleziono tekstu tego utworu",
+  "lyrics-loading-message": "Wczytywanie tekstu utworu...",
+  "lyrics-no-song-message": "Żaden utwór nie jest teraz odtwarzany",
+
+  "message-pausing-after-song": "Włączono wstrzymanie po tym utworze.",
+  "message-pausing-after-song-button": "Wyłączono wstrzymanie po tym utworze",
+  "message-uncaught-error": "Wystąpił niespodziewany błąd w aplikacji GPMDP.",
+  "message-uncaught-error-button": "Kliknij tutaj aby zgłosić ten błąd na stronie GitHub",
+
+  "modal-confirmTray-title": "Chcieliśmy tylko uprzedzić",
+  "modal-confirmTray-content": "Zamknięcie głównego okna aplikacji (czyli to, co właśnie próbowałeś zrobić) w rzeczywistości zminimalizuje odtwarzacz do systemowego paska narzędzi. Możesz zmienić to ustawienie w oknie Ustawień Aplikacji.",
+  "modal-confirmUpdate-title": "Dostępna aktualizacja",
+  "modal-confirmUpdate-content": "Hurra, wygląda na to, że dostępna jest nowa wersja Google Play Music Desktop Player. Ta aktualizacja powinna przynieść całe mnóstwo nowych funkcji i poprawek. Rekomendujemy zaktualizowanie aplikacji najszybciej jak to możliwe, żeby zapewnić najlepsze wrażenia z korzystania z niej.<br />Mamy nadzieję, że polubisz nową wersję!",

```

Comment:
Better: "Hurra, wygląda na to, że dostępna jest nowa wersja Google Play Music Desktop Player. Ta aktualizacja powinna przynieść całe mnóstwo nowych funkcji i poprawek. Rekomendujemy aktualizacje aplikacji najszybciej jak to możliwe, aby zapewnić najlepsze wrażenia korzystania z niej.<br />Mamy nadzieję, że polubisz nową wersję!". Changed "zaktualizowanie" to "aktualizacje" (it's better to prenounce).
 Change "żeby" to "aby" to remove duplicate write it. Removed "z" for the same manner.